src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,820 @@
+                                <property name="icon-name">info-symbolic</property>
+                                <property name="popover">popover_smaa_corner_rounding</property>
+                                <style>
+                                  <class name="flat"/>
+                                </style>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkSpinButton">
+                                <property name='numeric'>true</property>
+                                <property name='valign'>center</property>
+                                <property name="adjustment">
+                                  <object class="GtkAdjustment" id="smaa_corner_rounding">
+                                    <property name="step-increment">10</property>
+                                    <property name="upper">100</property>
+                                  </object>
+                                </property>
+                              </object>
+                            </child>
+                          </object>
+                        </child>
+                      </object>
+                    </child>
+                  </object>
+                </child>
+                <child>
+                  <object class="AdwExpanderRow" id="clut">
+                    <property name="show_enable_switch">True</property>
+                    <property name="title" translatable="yes">Color LookUp Table</property>
+                    <child>
+                      <object class="AdwPreferencesRow">
+                        <child>
+                          <object class="AdwActionRow">
+                            <property name="title" translatable="yes">Path</property>
+                            <child>
+                              <object class="GtkEntry" id="input_entry">
+                                <property name="valign">center</property>
+                                <property name="editable">False</property>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkMenuButton">
+                                <property name='valign'>center</property>
+                                <property name="tooltip-text" translatable="yes">Select</property>
+                                <property name="icon-name">info-symbolic</property>
+                                <property name="popover">popover_clut</property>
+                                <style>
+                                  <class name="flat"/>
+                                </style>
+                              </object>
+                            </child>
+                            <child>
+                              <object class="GtkButton" id="lut_file_path">
```

Comment:
Please don't reinvent the wheel here. Just copy the [launch options script picker](https://github.com/bottlesdevs/Bottles/blob/master/src/ui/dialog-launch-options.ui#L56) design.